hasher-matcher-actioner/webapp/src/pages/collaborations/AddCollaborationModal.tsx

```diff
@@ -0,0 +1,209 @@
+          onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+            onChange(e.target.value)
+          }>
+          <option>Open this select menu</option>
+          {type.possible_values?.map(possible_value => (
+            <option key={possible_value} value={possible_value}>
+              {possible_value}
+            </option>
+          ))}
+        </Form.Control>
+      );
+    } else {
+      // Must be set..
+      control = (
+        <PillBox
+          handleNewTagAdd={tag => {
+            if (!value) {
+              onChange([tag]);
+            } else if (value.indexOf(tag) === -1) {
+              onChange(value.concat([tag]));
+            }
+          }}
+          handleTagDelete={tag => {
+            if (value.indexOf(tag) !== -1) {
+              onChange(value.filter((x: string) => x !== tag));
+            }
+          }}
+          pills={value || []}
+          placeholder="Add values..."
+        />
+      );
+    }
+  }
+
+  return (
+    <Form.Group className="mb-3" controlId="">
+      <Form.Label>{name}</Form.Label>
+      {control}
+    </Form.Group>
+  );
+}
+
+export default function AddCollaborationModal({
+  show,
+  onHide,
+  didAdd,
+}: AddCollaborationModalProps): JSX.Element {
+  const [existingSchemas, setExistingSchemas] = useState<
+    Record<string, CollabSchema>
+  >({});
+  const [collabClassName, setCollabClassName] = useState<string>();
+  const [schema, setSchema] = useState<CollabSchema>();
+  const [fieldValues, setFieldValues] = useState<Record<string, any>>({});
+
+  const notifications = useContext(NotificationsContext);
+
+  // Load all existing schemas.
+  useEffect(() => {
+    fetchAllCollabSchemas().then(setExistingSchemas);
+  }, []);
+
+  const handleAdd = () => {
+    if (collabClassName === undefined) {
+      notifications.error({message: 'Must select a collab type.'});
+      return;
+    }
+
+    addNewCollab(collabClassName, fieldValues).then(() => {
+      setCollabClassName(undefined);
+      setSchema({});
+      setFieldValues({});
+      notifications.success({message: 'Added config'});
+      didAdd();
+    });
+  };
+
+  return (
+    <Modal show={show} onHide={onHide} size="lg">
+      <Modal.Header closeButton>
+        <Modal.Title>Add a new Collaboration Config</Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <Container>
+          <Row>
+            <Col>
+              <p>Select from the following collab class types.</p>
+              <p>
+                {Object.keys(existingSchemas).length === 0 ? (
+                  <p>
+                    No Signal Exchange APIs enabled. Go to{' '}
+                    <Link to="/settings/exchanges">Settings</Link> and add one.
+                  </p>
+                ) : (
+                  Object.entries(existingSchemas).map(
+                    ([className, thisSchema]) => (
+                      <Button
+                        key={className}
+                        variant="link"
```

Comment:
This doesn't make it super clear that this is selectable. Is there a selector type available (dropdown?)